src/Features/ERDDAP/waterLevel/chart/chartDisplay.tsx

```diff
@@ -0,0 +1,91 @@
+import { PlatformTimeSeries } from "Features/ERDDAP/types"
+import { converter } from "Features/Units/Converter"
+import { UnitSystem } from "Features/Units/types"
+import { DataTimeSeries } from "Shared/timeSeries"
+import { useParams } from "next/navigation"
+import { useEffect, useState } from "react"
+import { DatumSelector } from "../DatumSelector"
+import { LargeTimeSeriesWaterLevelChart } from "./largeTimeSeriesChart"
+
+interface ChartTimeSeriesDisplayProps {
+  dataset: DataTimeSeries
+  unitSystem: UnitSystem
+  timeSeries: PlatformTimeSeries
+  standardName: string
+}
+
+export const WaterLevelChartDisplay: React.FunctionComponent<ChartTimeSeriesDisplayProps> = ({
+  timeSeries,
+  dataset,
+  standardName,
+  unitSystem,
+}: ChartTimeSeriesDisplayProps) => {
+  const [floodThresholds, setFloodThresholds] = useState<any>()
+  const [datumOffset, setDatumOffset] = useState<number | undefined>()
+  const [title, setTitle] = useState<string>()
+  const params = useParams()
+  const dataConverter = converter(standardName)
+
+  const getDefaultTitle = () => {
+    if (!Object.keys(timeSeries.datum_offsets).length) {
+      return timeSeries.data_type.long_name
+    }
+  }
+
+  useEffect(() => {
+    if (timeSeries.flood_levels.length) {
+      const floodLevelsMap = timeSeries.flood_levels.reduce((acc, level, index) => {
+        if (!acc[level.name] && typeof datumOffset === "number") {
+          acc[level.name] =
+            level.name === "Major"
+              ? {
+                  minValue: dataConverter.convertToNumber(level.min_value, unitSystem) + datumOffset,
+                  maxValue: dataConverter.convertToNumber(level.min_value, unitSystem) + 1 + datumOffset,
+                }
```

Comment:
Great point. I'm wondering if we can keep the major plot band since, as I understand from Hannah, these are the standard ways of referencing flood impacts. But within this plot band, allow the inclusion of other localized flood levels for reference? So the 100 and 500 year flood thresholds would be included in the `Major` plotband?

Reply:
Yes, we will probably want to keep the band styling for minor/moderate/major, but there may be cases where major or moderate may not be known, so we'd have to treat the lower ones as the max.

Then any custom values we probably would want to draw with a labeled line rather than a band.